src/components/Settings.svelte

```diff
@@ -1,20 +1,58 @@
 <script>
+  import { beforeUpdate, afterUpdate } from 'svelte';
   import { Tabs, Tab, TabContent, MaterialApp } from 'svelte-materialify/src';
   import UISettings from './settings/UISettings.svelte';
   import FilterSettings from './settings/FilterSettings.svelte';
 
   export let isStandalone = false;
+  export let isResizing = false;
 
   const settings = [
     { name: 'Interface', component: UISettings },
     { name: 'Filters', component: FilterSettings }
   ];
+
+  function redrawSlider() {
+    const sliderElement = document.querySelector('.ltl-settings-slider');
```

Comment:
Instead of ``document.querySelector``, get a reference to the parent div of the tabs component using ``bind:this`` and use ``querySelector`` from that.

Reply:
done